### PR TITLE
New package: sdcc-3.7.0

### PR DESCRIPTION
**Pull request by lvitals**

### CODE REVIEW

File: srcpkgs/sdcc/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'sdcc'
+pkgname=sdcc
+version=3.7.0
+revision=1
+wrksrc="${pkgname}"
+build_style=gnu-configure
+makedepends="flex bison patch gputils boost-devel"
```
Comment (maxice8) [inline srcpkgs/sdcc/template]: path is part of the base-devel package and can be removed.

File: srcpkgs/sdcc/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+depends="bash libgcc boost"
```
Comment (maxice8) [inline srcpkgs/sdcc/template]: libgcc is detected by shlibs.

**Comment by lvitals** [comment]
@maxice8 the build ran for some architectures, for others I did not have the bison package

**Comment by maxice8** [comment]
flex and bison are things that are run on the host system, move them to hostmakedepends

**Comment by lvitals** [comment]
problem with the connection when running bootstrap

File: srcpkgs/sdcc/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'sdcc'
+pkgname=sdcc
+version=3.7.0
+revision=1
+wrksrc="${pkgname}"
+build_style=gnu-configure
+hostmakedepends="flex bison"
+makedepends="gputils boost-devel"
```
Comment (abenson) [inline srcpkgs/sdcc/template]: gputils should be in hostmakedepends?

**Comment by lvitals** [comment]
@maxice8 @abenson Please, can you check that everything is correct?

**Pull request merged**